doc/source/howtos/upgrade_guide.rst

```diff
@@ -53,53 +50,49 @@ error.
 
 .. seealso:: sections :ref:`encrypt-traffic`, :ref:`configure`
 
```

Comment:
I think we should also add `wsrep_sst_auth` removal changes here. PXC-8.0 will create an SST user internally for data transfer, so the user should remove `wsrep_sst_auth` from my.cnf before starting the node with 8.0 packages.
Detailed info is available in this blog post: https://www.percona.com/blog/2019/10/03/percona-xtradb-cluster-8-0-new-feature-wsrep_sst_auth-removal/

Reply:
Added to the steps.